### PR TITLE
DOCKER-130 Create an image used by static resources deployed to LXC

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by rotty3000** [comment]
@brianchandotcom can we merge this? I've added a demo. We can improve it incrementally. We need this image published for full lifecycle client extensions demo.

File: templates/lxc-static-resources/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+The following is the `Dockerfile` that will be provided with LXC extensions:
+
+```Dockerfile
+FROM liferay/lxc-static-resources:latest
+
+COPY static/ /resources/
+```
+
+This instructs that the `META-INF/resources` provided be made available to the Caddy web server.
```
Comment (gamerson) [inline templates/lxc-static-resources/README.md:13]:
```suggestion
This instructs that the `static/*` resources provided be made available to the Caddy web server.
```
```suggestion
This instructs that the `META-INF/resources` provided be made available to the Caddy web server.
```

Comment (rotty3000) [reply templates/lxc-static-resources/README.md]: thx, fixed!

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-docker/compare/30415c3bda...facadbae28

**Comment by brianchandotcom** [comment]
@rotty3000 @gamerson I merged this! I made a lot of tiny changes. See upstream. If I broke anything, please send me a follow up PR.

I changed it to /public_html instead of /resources

**Pull request closed without merging**